README.md

```diff
@@ -125,6 +133,26 @@ cmp.willShowPrivacyMessage()
     );
 ```
 
+### `cmp.willShowPrivacyMessageSync()`
+
+returns: `Boolean`
+
+Returns `true` if the CMP will show the initial privacy message (or is, or was), or `false` if not.
```

Comment:
```suggestion
Returns `true` if the CMP will show the initial privacy message, or `false` if not.
```

Reply:
how come?

Reply:
It’s the triple-“or” that’s confusing.

> `1` if `A` (or `B`, or `C`), or `0` if not.

Could we make it as below?

```suggestion
Returns `true` if the CMP has shown, is showing or will show the initial privacy message. Returns `false` otherwise.
```

I don’t think too strongly about this in any case.